packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -38,14 +39,33 @@ export default class UpdateConfigure extends EasCommand {
       '💡 The following process will configure your project to run EAS Update. These changes only apply to your local project files and you can safely revert them at any time.'
     );
 
-    await ensureEASUpdatesIsConfiguredAsync(graphqlClient, {
+    await ensureEASUpdateIsConfiguredAsync(graphqlClient, {
       exp,
       projectId,
       projectDir,
       platform,
     });
 
     Log.addNewLineIfNone();
-    Log.log(`🎉 Your app is configured to run EAS Update!`);
+    Log.log(`🎉 Your app is configured with EAS Update!`);
+    Log.newLine();
+    Log.log(`${chalk.bold('Next steps')}:`);
+    Log.newLine();
+    Log.log('Update a production build:');
+    Log.log(`1. Create a new build. Example: ${chalk.green('`eas build --profile production`')}.`);
+    Log.log('2. Make changes in your project.');
+    Log.log(`3. Publish an update. Example: ${chalk.green('`eas update --channel production`')}.`);
```

Comment:
This is the convention in this repo:
```suggestion
    Log.log(`1. Create a new build. Example: ${chalk.bold('eas build --profile production')}.`);
    Log.log('2. Make changes in your project.');
    Log.log(`3. Publish an update. Example: ${chalk.bold('eas update --channel production')}.`);
```

See 
<img width="583" alt="Screenshot 2022-12-08 at 09 36 06" src="https://user-images.githubusercontent.com/5256730/206398485-98545152-71a0-4b63-be48-f497abd6c0c1.png">